### PR TITLE
Update readme. Add quickstart tags. Remove deprecated V3 tags.

### PR DESCRIPTION
**Pull request by pragnagopa**
- Update Read Me to include dotnet-isolated 7.0 tags , remove V3 tags , Add quickstart tags
- Remove Powershell 7 from build and publish pipeline. Powershell 7.0 is not supported on V4
- MCR Docs update PR - https://github.com/microsoft/mcrdocs/pull/32024

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Linux amd64 Tags
 | `4-slim`                                 | Debian 11  |
 | `4-appservice` | Debian 11  |
 | `4-dotnet6-core-tools` |  Debian 11  |
+| `4-appservice-quickstart` | Debian 11 |
```
Comment (CooperLink) [inline README.md:23]: In my opinion we should consider not listing the quickstart images as they are equivalent to the -appservice images.  We don't want customers to arbitrarily pin to the quickstart images thinking they are getting some benefit from it. Rather have all customers targetting the same appservice images. Makes hotfixes, future changes easier.

File: host/4/publish.yml [hunk 2]
```diff
@@ -295,36 +295,6 @@ steps:
...
-
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-slim
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-appservice
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-appservice-quickstart
-
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-slim
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice
-      docker push $TARGET_REGISTRY/powershell:$(TargetVersion)-powershell7-appservice-quickstart
```
Comment (CooperLink) [inline host/4/publish.yml:322]: Would you like me to delete the existing powershell 7 images? Or keep them up to support customers who are still on them ?

**Review by CooperLink — APPROVED** [review]
2 comments.  Mostly looks good to me. I think we should not list the quickstart images as they are just renamed appservice images. I would prefer no more customers depending on those. So we can explore getting rid of that clutter in the future

File: host/4/publish.yml (hunk 2 above)
Comment (pragnagopa) [reply host/4/publish.yml:322]: unfortunately we cannot remove as the images are public but we can stop building them as it is not a supported version

File: README.md (hunk 1 above)
Comment (pragnagopa) [reply README.md:23]: quickstart images will not be listed on docker hub - https://hub.docker.com/_/microsoft-azure-functions but it is worth calling out all the images that are being built in this repo in the read-me for better visibility.

Comment (pragnagopa) [reply README.md:23]: Resolving the comment and merging the PR. @Cooperlink - lets sync up offline if you have more concerns

**Pull request merged**